src/Cabin/Bridge/Model/UserAccounts.php

```diff
@@ -39,6 +39,9 @@ class UserAccounts extends ModelGear
         (self::RECOVERY_TOKEN_BYTES * 4 / 3)
     );
 
+    /** @var Zxcvbn $zxcvbn */
```

Comment:
U wot mate?

Reply:
It's an actual class name :P https://github.com/bjeavons/zxcvbn-php